### PR TITLE
Fix for snapshot empty name validation

### PR DESCRIPTION
**Pull request by PravinRanjan10**
**What this PR does / why we need it**:
Fix for snapshot empty name validation

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
```

### CODE REVIEW

**Comment by sfzeng** [comment]
LGTM

File: pkg/api/util/db.go [hunk 1]
```diff
@@ -213,6 +213,11 @@ func CreateFileShareSnapshotDBEntry(ctx *c.Context, in *model.FileShareSnapshotS
 	}
 
 	//validate the snapshot name
+	if in.Name == "" {
+		errMsg := fmt.Sprintf("Snapshot name can not be empty. Please give valid snapshot name")
```
Comment (leonwanghui) [inline pkg/api/util/db.go]: Error msg should start with lower case

File: pkg/api/util/db.go [hunk 2]
```diff
@@ -213,6 +213,11 @@ func CreateFileShareSnapshotDBEntry(ctx *c.Context, in *model.FileShareSnapshotS
...
+		log.Error(errMsg)
+		return nil, errors.New(errMsg)
+	}
 	if strings.HasPrefix(in.Name, "snapshot") {
 		errMsg := fmt.Sprintf("Names starting 'snapshot' are reserved. Please choose a different snapshot name.")
 		log.Error(errMsg)
```
Comment (leonwanghui) [inline pkg/api/util/db.go]: Same here

**Pull request merged**